src/olympia/devhub/tasks.py

```diff
@@ -451,67 +445,50 @@ def annotate_webext_incompatibilities(results, file_, addon, version_string,
-    log.info('X-awagner check_for_api_keys_in_file : checking keys for users '
-             '%s' % users)
 
     keys = []
     for user in users:
         try:
-            log.info('X-awagner check_for_api_keys_in_file : Getting key for '
-                     'user %s' % user)
             key = APIKey.get_jwt_key(user_id=user.id)
-            log.info(u'X-awagner check_for_api_keys_in_file : Got key %s for '
-                     'user %s' % (key.key, user))
             keys.append(key)
         except APIKey.DoesNotExist:
-            log.info('X-awagner check_for_api_keys_in_file : Key for user %s '
-                     'does not exist' % user)
             pass
 
     if len(keys) > 0:
-        log.info('X-awagner check_for_api_keys_in_file : Trying to open zip '
-                 'at %s' % upload.path)
         zipfile = SafeZip(source=upload.path)
-        log.info('X-awagner check_for_api_keys_in_file : Getting filelist for '
-                 '%s' % zipfile)
-        for filepath in zipfile.filelist:
-            log.info('X-awagner check_for_api_keys_in_file : Reading file %s'
-                     % filepath.filename)
-            file_ = zipfile.read(filepath)
-            for key in keys:
-                log.info('X-awagner check_for_api_keys_in_file : Checking for '
-                         'key %s in file %s' % (key.key, filepath.filename))
-
-                if key.secret in file_.decode('unicode-escape'):
-                    log.info('X-awagner check_for_api_keys_in_file : '
-                             'Developer API key for user %s found in '
-                             'submission.' % key.user)
-                    if key.user == upload.user:
-                        msg = ugettext('Your developer API key was found in '
-                                       'the submitted file. To protect your '
-                                       'account, the key will be regenerated.')
-                    else:
-                        msg = ugettext('The developer API key of a coauthor '
-                                       'was found in the submitted file. To '
-                                       'protect your addon, the key will be '
-                                       'regenerated.')
-                    insert_validation_message(
-                        results, type_='error',
-                        message=msg, msg_id='api_key_detected',
-                        compatibility_type=None)
-
-                    # Revoke and regenerate after 2 minutes to allow the
-                    # developer to fetch the validation results
-                    log.info('X-awagner check_for_api_keys_in_file : Calling '
-                             'revoke routine')
-                    revoke_and_regenerate_api_key.apply_async(
-                        kwargs={'key_id': key.id}, countdown=120)
+        zipfile.is_valid()
+        for zipinfo in zipfile.info_list:
+            if zipinfo.file_size > 0:
```

Comment:
nit/suggestion - could be `>=` whatever the length is for private keys. 64 bytes?